logging/Limiting_Access_to_Logs.md

```diff
@@ -15,39 +17,86 @@ only for a specified namespace.
 each tenant administrator's access so that they can access the 
 log information only for their SAS Viya tenant.
 
-This access is controlled by a combination of Kibana tenant spaces and 
+This access is controlled by a combination of Kibana-tenant spaces and 
 Open Distro for Elasticsearch roles, back-end roles, and role mappings. 
 Scripts are provided to create everything that is needed for each 
-scenario.
+scenario. For details, see [Implement Access Controls for a Tenant or Namespace](Limiting_Access_to_Logs.md#implement_access).
+
+## <a name="logadm"></a>The Logadm User and Its Access Controls
+
+### Overview
+
+As of the 1.1.3 release, the following items are created during the deployment process:
+
+- a set of role-based access controls (RBACs)
+- the logadm user, which is linked to the new RBACs
```

Comment:
As a markdown formatting practice, I tend to like when things like user names or commands are in `fixed-width` font using backticks (`logadm` in this case).

There are many occurrences of `logadm` throughout the doc that would be changed if you agree with the change.

Reply:
I can make that change.